lib/generators/devise_security/install_generator.rb

```diff
@@ -4,7 +4,7 @@ module DeviseSecurity
   module Generators
     # Generator for Rails to create or append to a Devise initializer.
     class InstallGenerator < Rails::Generators::Base
-      LOCALES = %w[en es de fr it ja tr ru nl pt uk].freeze
+      LOCALES = %w[en es de fr it ja tr ru nl pt uk hi].freeze
```

Comment:
Keep these alphabetized (you'll see this when you merge in latest master).

Reply:
done